### PR TITLE
Finalize Release 35

### PR DESCRIPTION
**Pull request by TCP1231**

### CODE REVIEW

File: changelogs/api-management-service.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 A regular Azure API Management service update was started on . This release will continue to rollout through May 2023.
 
-### New features, fixes, and improvement
+### New features, fixes, and improvements
 
 1. The Azure AD authentication in Configuration API v2 has been updated.
```
Comment (mikebudzynski) [inline changelogs/api-management-service.md:9]: In points 1 & 2, can you explain what changed to the customers and how they can start using it? "has been updated" is not meaningful.

File: changelogs/api-management-service.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
 2. The custom hostname in Configuration API v2 has been updated.
```
Comment (mikebudzynski) [inline changelogs/api-management-service.md:10]: In point 4, you could say:

> We fixed an issue that might have prevented changing the organization or email address in the email notifications.

Comment (mikebudzynski) [inline changelogs/api-management-service.md:10]: In point 3, what does "disabled" mean? Does it have any effect on the customers' deployments?

**Comment by tomkerkhove** [comment]
We should also highlight this one - https://techcommunity.microsoft.com/t5/microsoft-defender-for-cloud/microsoft-bolsters-cloud-native-security-in-defender-for-cloud/ba-p/3801818

**Comment by tomkerkhove** [comment]
I created a seperate PR for Configuration API since we have to pull the changes since ARM spec is not live yet - https://github.com/Azure/API-Management/pull/38

**Review by tomkerkhove — COMMENTED** [review]
I created a seperate PR for Configuration API since we have to pull the changes since ARM spec is not live yet - https://github.com/Azure/API-Management/pull/38

File: changelogs/api-management-service.md (hunk 1 above)
Comment (tomkerkhove) [reply changelogs/api-management-service.md:9]: No longer relevant, see #38

File: changelogs/api-management-service.md (hunk 2 above)
Comment (tomkerkhove) [reply changelogs/api-management-service.md:10]: Not correct and longer relevant, see #38

**Comment by mikebudzynski** [comment]
> We should also highlight this one - https://techcommunity.microsoft.com/t5/microsoft-defender-for-cloud/microsoft-bolsters-cloud-native-security-in-defender-for-cloud/ba-p/3801818
> 
> Also, we have our API event tomorrow for which you might want to highlight some things.

We can use this link for the Defender for APIs launch announcement: https://aka.ms/DfAPIlaunch

File: changelogs/api-management-service.md [hunk 3]
```diff
@@ -2,15 +2,21 @@
 
 ## Release - API Management service: April 2023
 
-A regular Azure API Management service update was started on . This release will continue to rollout through May 2023.
+### Highlights
+
+1. [Workspaces for Azure API Management is now in public preview.](https://azure.microsoft.com/updates/public-preview-workspaces-in-azure-api-management/)
```
Comment (mikebudzynski) [inline changelogs/api-management-service.md:7]: Shouldn't it "are now" rather than "is now"?

**Review by tomkerkhove — APPROVED** [review]

File: changelogs/api-management-service.md [hunk 4]
```diff
@@ -2,15 +2,21 @@
...
+2. [Incoming traffic to Azure API Management's gateway can be secured to clients running in a virtual network through Azure Private Link.](https://azure.microsoft.com/updates/generally-available-azure-private-link-support-in-azure-api-management/)
+3. [Microsoft bolsters cloud-native security in Defender for Cloud with new API security capabilities](https://techcommunity.microsoft.com/t5/microsoft-defender-for-cloud/microsoft-bolsters-cloud-native-security-in-defender-for-cloud/ba-p/3801818)
```
Comment (mikebudzynski) [inline changelogs/api-management-service.md:9]: 3 and 4 are the same. Could you change them to this:

`3. [Azure API Management and Microsoft Defender for APIs integration is now in preview](https://azure.microsoft.com/updates/public-preview-azure-api-management-and-microsoft-defender-for-apis-integration/). [Learn how Microsoft bolsters cloud-native security in Defender for Cloud with new API security capabilities](https://aka.ms/DfAPIlaunch).`

**Comment by tomkerkhove** [comment]
Some new release:
- https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/Container-v2.2.0
- https://github.com/Azure/api-management-self-hosted-gateway/releases/tag/v1.6.0

**Review by mikebudzynski — CHANGES_REQUESTED** [review]

**Review by TCP1231 — COMMENTED** [review]
Changes complete.

**Review by mikebudzynski — APPROVED** [review]

**Pull request merged**